src/foam/dao/CSVSink.js

```diff
@@ -0,0 +1,176 @@
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'CSVSink',
+  extends: 'foam.dao.AbstractSink',
+
+  documentation: `Sink runs the csv outputter, and contains the resulting string in this.csv`,
```

Comment:
Only one line, just just use regular quotes, which are faster: '

Reply:
I'll be more careful with this in the future. Sorry about that. Good catch.